core-registry-ui/defaults/main.yml

```diff
@@ -21,3 +21,15 @@ basic_authentication: false
 
 # True or false if Nginx will be proxying the CADT API
 core_registry_api_proxy: true
+
+# Default to blank which will set UI to not autopopulate host field.
+# Override in playbook with full URL including path and scheme.
+cadt_api_url: ""
+tokenization_api_url: ""
+explorer_api_url: ""
```

Comment:
these are probably pretty specific var names, but.. should they be prefixed with cadt or something anyways?